ISP-BCC.md

```diff
@@ -15,6 +15,18 @@ Base choice coverage is a testing criterion that focuses on testing the various
 |        | server.Name (string)       | | | |
 
 
+### checkTrustDomain(trustDomain string)
```

Comment:
Not for now, but have you considered using the standard [godoc](https://www.makeuseof.com/go-godoc-document-code-automatically/) for documenting methods? It's usually easier to maintain documentation if it sits next to the code, and you could also generate a website from it.

Reply:
Have not considered it because I did not know it existed! I will have to look into it! 😊 thank you for the suggestion!